### PR TITLE
Fetch the last_message from conversation instead of last read message

### PR DESCRIPTION
**Pull request by rickmak**
connect SkygearIO/chat#76

### CODE REVIEW

File: chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java [hunk 1]
```diff
@@ -659,9 +661,10 @@ public void onSucc(@Nullable List<Message> messages) {
                                 mMap.put(m.getId(), m);
                             }
                             for (UserConversation uc: userConversations) {
-                               if (uc.getLastReadMessageId() != null) {
-                                   uc.lastMessage = mMap.get(uc.getLastReadMessageId());
-                               }
+                                Conversation c = uc.conversation;
+                                if (c.getLastMessageId() != null) {
+                                    uc.lastMessage = mMap.get(c.getLastMessageId());
+                                }
```
Comment (ben181231) [inline chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java]: @rickmak Do we still need to fetch the last **read** message of the user conversation ?

Comment (ben181231) [inline chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java]: And I also found that the `UserConversation.getLastReadMessage()` will return the internal property `lastMessage`, which I think it should be wrong in naming.

Comment (rickmak) [inline chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java]: @ben181231 let skip last _read_ message transient include for now. Seem it is not worth to add extra param for it. Seem none is requesting and no use case.

**Comment by rickmak** [comment]
@ben181231 updated.

File: chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java (hunk 1 above)
Comment (ben181231) [inline chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java]: Okay.

**Pull request merged**